samples/snippets/quickstart_analyzeiampolicy.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+from google.cloud import asset_v1
+
+
+def build_analysis_query(parent):
```

Comment:
If you don't have a strong reason to have this helper function, can you inline all the code in `analyze_iam_policy`?

```python
    # [START asset_quickstart_analyze_iam_policy]

    # TODO project_id = 'Your Google Cloud Project ID'

    client = asset_v1.AssetServiceClient()
    parent = "projects/{}".format(project_id)
    analysis_query = asset_v1.IamPolicyAnalysisQuery()
    analysis_query.scope = parent
    analysis_query.resource_selector.full_resource_name = f"//cloudresourcemanager.googleapis.com/{parent}"
    analysis_query.options.expand_groups = True
    analysis_query.options.output_group_edges = True
    response = client.analyze_iam_policy(
        request={"analysis_query": analysis_query}
    )
    print(response)
    # [END asset_quickstart_analyze_iam_policy]
```

Reply:
Done